CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+## How to Create an Issue (first iteration):
+# ![Subspace](subspace-cover.jpg)
+
+## Contributing to Subspace
+
+Thank you for considering to help out with the source code! Subspace welcomes help in many forms including development, code review, documentation improvements, and outreach. We are grateful for even the smallest of improvements!
+
+> [I don't want to contribute, I just have some questions?](#I-dont-want-to-contribute-I-just-have-some-questions)
+
+We have a [Code of Conduct](./CODE_OF_CONDUCT.md), please follow it in all interactions with project maintainers and fellow users :)
+
+## Issues
+
+You want to write an issue if you have a
+- feature request
+- bug report
+- technical question
+
+### How to Create an Issue?
+
+- Summarize the issue briefly in the issue title
+- If it is a bug:
+    - Explain, what is the expected behavior
+    - Explain, what happened instead
+    - Submit the log file
+- If it is a feature request:
+    - Clearly explain the feature that is being requested
+    - Explain why do you think we need this feature / how would it enhance the application
+
+**Where is my log file?**
+Below are the `log` file locations (replace `alice` with your username)
+- windows: `C:\Users\Alice\AppData\Roaming\com.subspace.desktop`
+- macos: `/Users/Alice/Library/Logs/com.subspace.desktop`
+- linux: `/home/alice/.config/com.subspace.desktop`
```

Comment:
What are logs doing in `.config` on Linux? They should be in `.local/share` instead. `.config` is for configuration.

Reply:
I'm using the default folder choices from `tauri-plugin-log`. Here it is:
https://github.com/tauri-apps/tauri-plugin-log/blob/dev/src/lib.rs#L106

I can open an issue on there to suggest `.local/share` instead.

Reply:
Btw, the convention for linux seems to be `var/log`. Is there a reason you prefer `.local/share` instead of `var/log`?

Reply:
Please do, I think it is really important to have logs in the correct location.

No, `/var/log` is for system stuff, regular users don't have access there, and even for system stuff it is generally preferred to go through things like systemd.